libs/common/src/lib/search-widget/search-widget.models.ts

```diff
@@ -275,7 +275,7 @@ export function getRagStrategies(ragStrategiesConfig: RagStrategiesConfig) {
 
   return { ragProperties, ragImagesProperties };
 }
-export function getNotEnoughDataMessage(config: GenerativeAnswerConfig): string {
+export function getNotEnoughDataMessage(config: WidgetConfiguration): string {
```

Comment:
It seems the getNotEnoughDataMessage still doesn't work with all LLM. But I think we cannot fix it in the front.